spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -198,6 +200,15 @@ public Table display(
 		return builder.build();
 	}
 
+	@CliCommand(value = TASK_EXECUTION_CLEANUP, help = "Clean up any platform specific resources linked to a task execution")
+	public String cleanup(
+		@CliOption(key = { "", "id" },
+			help = "the task execution id",
+			mandatory = true) long id) {
+		taskOperations().cleanup(id);
+		return String.format("Cleaned up resources for task execution %s", id);
```

Comment:
I guess the `cleanup` may be synchronous/asynchronous based on the implementation. Can we have the return message something like `Cleanup request for task execution <id> is submitted`